tests/scripts/remote_monitoring_tests/rest_apis/test_e2e_workflow.py

```diff
@@ -81,32 +80,26 @@ def test_list_recommendations_multiple_exps_from_diff_json_files(cluster_type):
 
             # Expecting that we have recommendations
             if j > 96:
-                # Sleep for 1 sec to get recommendations
-                time.sleep(1)
-
+                response = update_recommendations(experiment_name, interval_start_time, end_time)
```

Comment:
Please update all the tests in test_list_recommendations.py to invoke update_recommendations

Reply:
done